klarna-on-demand-android.iml

```diff
@@ -1,21 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where'd this file go? Is this one of those damn IDE files that keep popping in and out?